composition/CMakeLists.txt

```diff
@@ -190,6 +194,13 @@ if(BUILD_TESTING)
   endif()
 
   call_for_each_rmw_implementation(tests)
+
+  find_package(ament_cmake_flake8 REQUIRED)
```

Comment:
is this necessary ? this package already find_package the `ament_lint_auto` dependencies.
If we want the ability to make some of them `REQUIRED` we could implement something similar to https://github.com/ament/ament_cmake/pull/93 in ament_lint_auto and just pass ament_cmake_flake8 as a required dependency

Reply:
I am just applying "find what you need". Since the code is using a CMake function of that package it is finding it explicitly.